common/services/allocation.js

```diff
@@ -8,6 +8,27 @@ function urlKeyToFilter(additionalFilters) {
   })
 }
 const allocationService = {
+  format(data) {
+    const booleansAndNulls = ['complete_in_full', 'sentence_length']
```

Comment:
I'm not suggesting that we do anything about it as part of this PR - you are stepping in the footsteps of those who went before, but maybe we should consider figuring out what kind of formatting the field needs from its definition. Of course, there is a bit of a disjoint between the model and fields so hmmm...

Reply:
Hmm... please explain. Are you suggesting we use some kind of Hungarian notation? Or just document the format required in the FE?

Reply:
Not hungarian notation. Not just documenting. Adding a property to the fields that enables it to derived automatically and not have to update the format methods manually

```js
fields: {
  complete_in_full: {
    format: 'boolean',
    ...
  },
  an_other: {
    format: 'identifier'
  ...
  }
...
```

Reply:
It's a good idea @solidgoldpig .